apps/home/templates/account/base.html

```diff
@@ -2,7 +2,7 @@
 {% load crispy_forms_tags %}
 
 {% block content %}
-<h1 class="row justify-content-center mt-2">{% block title %}{% endblock %}</h1>
```

Comment:
I kind of liked how it looked centred. Or is it better to stay with the standard alignment?

Reply:
Ah, I missed that. Should be fixed in c8bb567 .